.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: node_js
 node_js:
-  - '4'
-
+  - '6'
+  - '8'
+  - '10'
```

Comment:
Can we change this to just 8 and 10 please? I think these versions should be plenty.